mibidata/tests/test_tiff.py

```diff
@@ -16,7 +16,7 @@
 from mibidata import mibi_image as mi
 from mibidata import tiff, util
 
-DATA = np.arange(500).reshape(10, 10, 5).astype(np.uint16)
+DATA = np.arange(500).reshape(10, 10, 5).astype(np.float)
```

Comment:
Change this to `np.float32` which will require some changes below too.

Reply:
Done.